src/test/resources/ages-alternative-malformed.csv

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Please include one malformed line after the header as well. To have better coverage, please consider other types of malformed lines (like missing only one column or having extra?)